_posts/en/newsletters/2022-05-11-newsletter.md

```diff
@@ -24,14 +24,39 @@ relevant.
 meeting, highlighting some of the important questions and answers.  Click on a
 question below to see a summary of the answer from the meeting.*
 
-FIXME:glozow is a PR by FIXME to FIXME.
+[Improve Indices on pruned nodes via prune blockers][bitcoin core 21726] is a PR by Fabian Jahr to
+introduce a new method for deciding when it is safe to prune a node from block storage. This new
+method enables pruning nodes to maintain a Coinstats index and removes the validation module's
+dependency on index-related code.
 
 {% include functions/details-list.md
 
-  q0="FIXME"
-  a0="FIXME"
-  a0link="https://bitcoincore.reviews/FIXME#l-17"
+  q0="What indexes currently exist in Bitcoin Core, and what do they do?"
+  a0="A node may maintain up to three optional indexes to help efficiently retrieve data from disk.
+The transaction index (`-txindex`) maps transaction hashes to the block in which transaction can be
+found. The block filter index (`-blockfilterindex`) indexes BIP157 block filters for each block.
+The coin stats index (`-coinstatsindex`) stores statistics on the UTXO set."
+  a0link="https://bitcoincore.reviews/21726#l-28"
 
+  q1="What is a circular dependency? Why do we want to avoid them when possible?"
+  a1="A circular dependency between two code modules exists when both cannot be used without the
+other. While circular dependencies are not a security issue, they signify poor code organization
+and encumber development by making it more difficult to build, use, and test specific modules or
+functionality in isolation."
+  a1link="https://bitcoincore.reviews/21726#l-44"
+
+  q2="How do the prune blockers introduced in [this commit][review club commit] work?"
```

Comment:
@glozow: Are "prune blockers" and "prune locks" the same things? If so, let's use only one term.

Reply:
> @glozow: Are "prune blockers" and "prune locks" the same things? If so, let's use only one term.

IIRC the "prune blockers" naming was replaced in a later push by prune locks.